firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/internal/ReleaseIdentificationUtils.java

```diff
@@ -90,20 +90,23 @@ public static String calculateApkHash(@NonNull File file) {
       zipFingerprint = sb.toString();
 
     } catch (IOException | NoSuchAlgorithmException e) {
-      Log.v(TAG, String.format("id calculation failed for %s", file.getPath()));
+      LogWrapper.getInstance().v(TAG + "id calculation failed for " + file.getPath());
       return null;
     } finally {
       long elapsed = System.currentTimeMillis() - start;
       if (elapsed > 2 * 1000) {
-        Log.v(
-            TAG,
-            String.format(
-                "Long id calculation time %d ms and %d entries for %s",
-                elapsed, entries, file.getPath()));
+        LogWrapper.getInstance()
+            .v(
+                TAG
+                    + String.format(
+                        "Long id calculation time %d ms and %d entries for %s",
+                        elapsed, entries, file.getPath()));
       }
 
-      Log.v(TAG, String.format("Finished calculating %d entries in %d ms", entries, elapsed));
-      Log.v(TAG, String.format("%s hashes to %s", file.getPath(), zipFingerprint));
+      LogWrapper.getInstance()
+          .v(TAG + String.format("Finished calculating %d entries in %d ms" + entries, elapsed));
```

Comment:
should this be a comma instead of a + sign?

Reply:
YES! great catch